content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+| Network rules or VPN prevent the Browser RUM SDK from being downloaded or from sending the data to Datadog. | You may need to grant access to the endpoints required to download the SDK or to send the data. The list of endpoints is available in our [CSP documentation][6].                                        |
+
+You may also want to check our [Content Security Policy guidelines][6] to make sure your website successfully grants access to the Browser RUM SDK CDN and the intake endpoint.
+
+### Check that the Browser RUM SDK is initialized
+
+Run the `window.DD_RUM.getInternalContext()` command in your browser console and check that an `application_id`, `session_id`, and view object are returned:
+
+{{< img src="real_user_monitoring/browser/troubleshooting/success_rum_internal_context.png" alt="Successful get internal context command">}}
+
+If the SDK is not installed, or if it is not successfully initialized, you may see the `ReferenceError: DD_RUM is not defined` error like the one below:
+
+{{< img src="real_user_monitoring/browser/troubleshooting/error_rum_internal_context.png" alt="Error get internal context command">}}
+
+You can also check in your browser console if you notice any errors related to the loading of the Browser RUM SDK.
+
+### Check that data is sent to the Datadog intake
+
+The Browser RUM SDK sends batches of data periodically to the Datadog intake. You should see network requests targeting `/v1/input` (the URL origin part may differ due to RUM configuration) in the Network section of your browser Developer Tools:
+
+{{< img src="real_user_monitoring/browser/troubleshooting/network_intake.png" alt="RUM requests to Datadog intake">}}
+
+## RUM Cookies
+
+The Browser RUM SDK relies on cookies to group all events generated from a unique [user session][7]. The cookies are 1st party (they are set on your domain) and are not used for cross-site tracking. Here are the cookies set by the Browser RUM SDK:
+
+| Cookie name        | Details                                                                                                                                                                                                                                                                                                  |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `_dd_s`             | Session cookie, used to group all events generated from a unique user session across multiple pages. It is extended for an extra 15 minutes when the user interacts with the website, in the limit of the maximum user session duration of 4 hours. It also contains whether the session should be sampled. |
```

Comment:
Calling this the "Session cookie" may be a bit confusing, because this can also designate [cookies that expires when the browser is closed](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_the_lifetime_of_a_cookie).

```suggestion
| `_dd_s`             | Cookie used to group all events generated from a unique user session across multiple pages. It is extended for an extra 15 minutes when the user interacts with the website, in the limit of the maximum user session duration of 4 hours. It also contains whether the session should be sampled. |
```